src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -489,6 +500,9 @@ void unpack_i4f16(const ov::SoPtr<ov::ITensor>& from,
             pDst[i] = tmp[i];
         }
     }
+#else
+    throw std::runtime_error("AVX2 support is neccessary but it's not enabled!");
```

Comment:
why not `NPUW_ASSERT` ?

Reply:
Thanks for suggestion, fixed, but decided to use OPENVINO_THROW (as don't need to check any expression here)